merlin/legacy/inference/triton/data_conversions.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import itertools
+
+try:
+    import cudf
+    import cupy as cp
+except ImportError:
+    cudf = cp = None
+
+import numpy as np
+import pandas as pd
+
+from merlin.core.dispatch import build_cudf_list_column, is_list_dtype
+from merlin.dag import Supports
+
+
+def convert_format(tensors, kind, target_kind):
```

Comment:
Looks like we already have most (all?) of the functions from this file in [merlin.systems.triton.conversions](https://github.com/NVIDIA-Merlin/systems/blob/v0.8.0/merlin/systems/triton/conversions.py#L42)

Reply:
We do, but this is intended to be a direct copy/paste into a standalone package that's easy to delete. Since this is for maintaining legacy support, I don't want to consolidate the repetitive functions, because changing the versions we're actively using in Systems should be okay but changing these may break legacy functionality that has limited tests.